dnf-behave-tests/dnf/install-file-conflicts.feature

```diff
@@ -0,0 +1,17 @@
+@dnf5
+Feature: Tests for file conflicts reporting
+
+Background:
+  Given I use repository "dnf-ci-install-conflicts"
+
+
+Scenario: An error is reported when a package with file conflicts is tried to be installed
+   When I execute dnf with args "install FileConflict-1.0-1.x86_64"
+   Then the exit code is 0
+    And Transaction is following
+        | Action                    | Package                      |
+        | install                   | FileConflict-0:1.0-1.x86_64  |
+   When I execute dnf with args "install FileConflict-2.0-1.x86_64"
```

Comment:
I find this quite confusing. It is not actually an `install` its doing an `upgrade` of pkg `FileConflict` from version 1 to 2.

I am also not sure why the conflict even happens? Is it because the `a_dir` changes from a file to a directory? Why is that a problem though? The packages are not installed at the same time.

For the test it would be much clearer to me if we had a `pkg-a` and `pkg-b` both containing the same file.

Reply:
Yes, it could be simplified. I basically copied that from a very similar use case in the `module/install-errors.feature`. But I can create a simpler test case.

Reply:
I would prefer simplification.

Actually this is an RPM issue: https://docs.fedoraproject.org/en-US/packaging-guidelines/Directory_Replacement/  it would be better not to rely on it.

Reply:
Oh, OK. Thanks for the context, then it definitely deserves rewriting.